### ISSUE TITLE
Reproduce the dice result on MSD

### ISSUE THREAD

**Issue by MelindaDong** [zhaoziheng/SAT#27]
Hi, 
I am working with MSD challenge, and I'd like to reproduce the reported result.

1. How do you report the multi-channel and multi-class task?  For example, MSD Prostate(t2 and adc) with  2 classes(pz and tz), did you separate each modality and report as the average dice?
2. For  MSD, It has it's own train test set, is the reported dice from the online evaluation of the original test set? Or do you have your own random train-test split on the original training set?

<img width="1154" height="454" alt="Image" src="https://github.com/user-attachments/assets/0ce48d87-8e19-4d69-b46d-d6f7bc821b3a" />

**Comment by MelindaDong** [comment]
How did you evaluate the BraTS data?  I noticed you mentioned we need to separate each modality(t1,t2,t1ce, flair), just confirm if the final dice reported is an average of 4 modalities or just use t1ce(since it is the best contrast for brain tumor)?

**Comment by zhaoziheng** [comment]
1. Yes. Each modality is treated as an individual sample in evaluation. 
2. The whole evaluation is conducted offline. So we randomly re-split the training sets in the MSD challenge into training and test sets. The split is at https://github.com/zhaoziheng/SAT-DS/tree/main/data/split_json.
3. For BraTS, we use all the modalities for training and evaluation.
